switchboard/dds_util.py

```diff
@@ -255,7 +256,7 @@ def get_context(self):
         project = self.get_project()
         project_url = self.get_project_url()
         return {
-            'service': 'Duke Data Service',
+            'service_name': 'Duke Data Service',
```

Comment:
Duplicate 'Duke Data Service' string in this file. Also different than how this is handled in `s3_util.py`  where it uses `base_context['service_name']`. Or perhaps just a global constant would suffice. Also it would be nice to test this new functionality since we are already doing it for the s3 version.